### PR TITLE
Most recent strategist runlogs

### PR DESCRIPTION
**Pull request by DanielVF**
History of governance actions and strategist actions I've created over the last few months. Should have been committing weekly after I did them.

### CODE REVIEW

File: brownie/addresses.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 COMP_STRAT = '0x9c459eeb3FA179a40329b81C1635525e9A0Ef094'
 CONVEX_STRAT = '0xEA2Ef2e2E5A749D4A66b41Db9aD85a38Aa264cb3' 
 FLIPPER = '0xcecaD69d7D4Ed6D52eFcFA028aF8732F27e08F70'
-BUYBACK = '0x7294cd3c3eb4097b03e1a61eb2ad280d3dd265e6'
```
Comment (DanielVF) [inline brownie/addresses.py:19]: Buyback changed during an upgrade

File: brownie/addresses.py [hunk 2]
```diff
@@ -33,4 +33,6 @@
 
 # Other
 UNISWAP_V2_ROUTER = '0x7a250d5630b4cf539739df2c5dacb4c659f2488d'
-OUSD_USDT = '0xcc01d9d54d06b6a0b6d09a9f79c3a6438e505f71'
\ No newline at end of file
+OUSD_USDT = '0xcc01d9d54d06b6a0b6d09a9f79c3a6438e505f71'
+
+OUSD_3CRVF_POOL = '0x87650D7bbfC3A9F10587d7778206671719d9910D'
```
Comment (DanielVF) [inline brownie/addresses.py:38]: Added the meta pool address

File: brownie/allocations.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ def load_from_blockchain():
     base = pd.DataFrame.from_records(
         [
             ["AAVE", "DAI", int(world.aave_strat.checkBalance(world.DAI) / 1e18)],
-            # ['AAVE','USDC', int(world.aave_strat.checkBalance(world.USDC)/1e6)],
+            ['AAVE','USDC', int(world.aave_strat.checkBalance(world.USDC)/1e6)],
```
Comment (DanielVF) [inline brownie/allocations.py:22]: We now have AAVE USDC enabled, so we can report on it.

File: brownie/allocations.py [hunk 4]
```diff
@@ -199,17 +199,3 @@ def transactions_for_reallocation(votes):
     txs = wrap_in_loss_prevention(txs, vault_loss_raw)
 
     return txs
-
-
```
Comment (DanielVF) [inline brownie/allocations.py:203]: No need for an example at the bottom of the file, now that we have the run logs.

**Review by sparrowDom — APPROVED** [review]
Confirmed all the addresses, LGTM

**Pull request merged**